### ISSUE TITLE
Feature request: `rename_all` container attribute

### ISSUE THREAD

**Issue by allan2** [sfackler/rust-postgres#934]
A Postgres type like so
```sql
CREATE TYPE mood AS ENUM ('sad', 'happy');
```

can be represented like this
```rust
#[derive(ToSql, Debug)]
#[postgres(name = "mood")]
enum Mood {
    #[postgres(name = "sad")]
    Sad,
    #[postgres(name = "happy")]
    Happy
}
```

This is a request for a container attribute like Serde's `rename_all` <sup>[[1]](https://serde.rs/container-attrs.html)

```rust
#[derive(ToSql, Debug)]
#[postgres(rename_all = "lowercase")]
enum Mood {
    Sad,
    Happy
}
```

What do you think?

--
#894 related

**Comment by sfackler** [comment]
Seems like a reasonable feature to me!

**Comment by jakubadamw** [comment]
@allan2 @sfackler I made an attempt at an initial implementation in #952. Feedback would be welcome!

**Comment by allan2** [comment]
Closing because merged in #1008 and released in postgres-types 0.2.6. Thanks @JaydenElliott @sfackler!
